### PR TITLE
implemented database tables

### PR DESCRIPTION
**Pull request by lucycpu**
I did the work, please review and let me know if anything needs to be changed

### CODE REVIEW

**Comment by lucycpu** [comment]
Hey @charliecpu I see now that the reports were left out of the spec, please add them in

File: 1 tables.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+create database TrackGermDB
```
Comment (charliecpu) [inline 1 tables.sql:5]: I was unable to run the file more than once. I think you may be missing a `drop db statement`.

File: 1 tables.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+create database TrackGermDB
+go
+use TrackGermDB
+go
+create table dbo.germ(
```
Comment (charliecpu) [inline 1 tables.sql]: missing a primary key

File: 1 tables.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
+create database TrackGermDB
+go
+use TrackGermDB
+go
+create table dbo.germ(
+	GermName varchar(200) not null,
+	Host varchar(20)
```
Comment (charliecpu) [inline 1 tables.sql]: this should not allow `null`, also we need a check constraint as is in the spec

File: 2 sample data.sql [hunk 4]
```diff
@@ -0,0 +1,6 @@
+select 'Norovirus', 'human'
```
Comment (charliecpu) [inline 2 sample data.sql:6]: missing the insert statement

**Review by charliecpu — CHANGES_REQUESTED** [review]
I reviewed the work, pretty good. I noticed a few things missing, please my inline comments.

File: 1 tables.sql (hunk 1 above)
Comment (lucycpu) [reply 1 tables.sql:5]: added commit to fix these problems

File: 1 tables.sql (hunk 2 above)
Comment (lucycpu) [reply 1 tables.sql]: see previous, my latest commit addresses this also

File: 1 tables.sql (hunk 3 above)
Comment (lucycpu) [reply 1 tables.sql]: added the not null

File: 2 sample data.sql (hunk 4 above)
Comment (lucycpu) [reply 2 sample data.sql:6]: I added the insert statement

**Review by lucycpu — COMMENTED** [review]
Thanks for the feedback, I addressed all the issues you raised, see my latest commits.

File: 1 tables.sql (hunk 2 above)
Comment (charliecpu) [reply 1 tables.sql]: all good

File: 1 tables.sql (hunk 1 above)
Comment (charliecpu) [reply 1 tables.sql:5]: thanks all good

File: 2 sample data.sql [hunk 5]
```diff
@@ -0,0 +1,8 @@
+delete germ
```
Comment (charliecpu) [inline 2 sample data.sql:3]: missing the `use` statement

File: 2 sample data.sql [hunk 6]
```diff
@@ -0,0 +1,8 @@
+delete germ
+
+insert germ(GermName, Host)
+select 'Norovirus', 'human'
+select 'Salmonella', 'plant'
```
Comment (charliecpu) [inline 2 sample data.sql]: this insert only inserted 1 row... is _something_ missing?

**Review by charliecpu — CHANGES_REQUESTED** [review]

File: 1 tables.sql (hunk 3 above)
Comment (lucycpu) [reply 1 tables.sql]: this is done, I am resolving the conversation

File: 2 sample data.sql (hunk 4 above)
Comment (lucycpu) [reply 2 sample data.sql:6]: this is done in latest commit

File: 2 sample data.sql (hunk 5 above)
Comment (charliecpu) [reply 2 sample data.sql:3]: this is good, thanks

File: 2 sample data.sql (hunk 6 above)
Comment (charliecpu) [reply 2 sample data.sql]: all good, thank you

File: 2 sample data.sql (hunk 4 above)
Comment (charliecpu) [reply 2 sample data.sql:6]: all good!

**Review by charliecpu — APPROVED** [review]

**Pull request closed without merging**